src/main/java/com/neva/felix/webconsole/plugins/search/core/OsgiExplorer.java

```diff
@@ -61,28 +62,50 @@ public OsgiExplorer(BundleContext context) {
 	}
 
 	public File findDir(Long bundleId) {
-		return new File(String.format(BUNDLE_PATH_FORMAT, context.getProperty(BUNDLE_STORAGE_PROP), bundleId));
+		String bundleStorage = context.getProperty(BUNDLE_STORAGE_PROP);
```

Comment:
This line returns the config value from felix configuration.
In my `config.properties` file i have an entry of: `${felix.cache.rootdir}/felix-cache/bundles`.
Then `bundleStorage == "/felix-cache/bundles"` and i think that on windows this is treated as absolute path.
When I don't have this property configured then felix uses default folder: `felix-cache`, but the property is empty.

At the end i have used the configuration of:
```
org.osgi.framework.storage=./felix-cache/bundles
```
but something is still wrong :?